test/spec/modules/eids_spec.js

```diff
@@ -238,6 +238,40 @@ describe('eids array generation for known sub-modules', function() {
     });
   });
 
+
+  it('sovrn', function() {
+    const userId = {
+      sovrn: {'id': 'sample_id'}
+    };
+    const newEids = createEidsArray(userId);
+    expect(newEids.length).to.equal(1);
+    expect(newEids[0]).to.deep.equal({
+      source: 'liveintent.sovrn.com',
+      uids: [{
+        id: 'sample_id',
+        atype: 3
+      }]
+    });
+  });
+
+  it('sovrn with ext', function() {
+    const userId = {
+      sovrn: {'id': 'sample_id', 'ext': {'provider': 'some.provider.com'}}
```

Comment:
```suggestion
      sovrn: {'id': 'sample_id', 'ext': {'provider': 'liveintent.sovrn.com'}}
```

Reply:
That should be fine as is. The goal of the test is to check if the given `ext.provider` will be used to generate the EID. In the test we provide 

```
'ext': {'provider': 'some.provider.com'}
```

thus we expect the same provider in resulting EID.

Reply:
Got it, thank you.